plugins/com.google.cloud.tools.eclipse.googleapis/src/com/google/cloud/tools/eclipse/googleapis/IGoogleApiFactory.java

```diff
@@ -37,6 +37,8 @@ public interface IGoogleApiFactory {
    */
   public Account getAccount() throws IOException;
   
+  public boolean isLoggedIn();
```

Comment:
Temporary as better approach is used in https://github.com/GoogleCloudPlatform/google-cloud-eclipse/pull/3713